babel-watch.js

```diff
@@ -222,43 +222,42 @@ function generateTempFilename() {
   ].join(''));
 }
 
-function handleFileLoad(filename, callback) {
+async function handleFileLoad(filename) {
```

Comment:
Are you sure we can turn the entire system synchronous?
I'm concerned about this comment inside runner.js:
<img width="833" alt="image" src="https://github.com/kmagiera/babel-watch/assets/18382880/8a761b68-9317-419b-9f85-31fae037f3c4">

Reply:
The loader itself is still synchronous, which is why it makes the pipes synchronously. The function handling the load itself was always async, it just was async with callbacks. Now it's async with promises.

Reply:
Okay, I see, it was hybrid sync & async, since babel.transformFile is async.
And it doesn't really matter if we make parent process even more async, since runner is kept synchronous and is forced to wait for the synchronous pipe.